Guideline/FIRST_RUN.md

```diff
@@ -0,0 +1,379 @@
+# First time developer guideline
```

Comment:
Why did you move the guide line file since there is only one file in the folder?

Reply:
I want to make the guidelines better by separating them according to the function of each guideline to make it easier to access and read in the next task.